### ISSUE TITLE
Add alt tag to maker shadow

### ISSUE THREAD

**Issue by topicus** [Leaflet/Leaflet#5258]
There is an old PR (https://github.com/Leaflet/Leaflet/pull/2112) that addressed alt text for icons. However in order to pass accessibility tests the same needs to happen with maker shadows. 

I'm wondering if it's ok just set a default alt text for icons.

## How to reproduce
- Leaflet version:  1.0.2
- Browser: Chrome 55
- OS/Platform (with version) I'm using: MacOS Sierra
- Create a map with default markers
- Run wave accessibility tool or similar (https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
- A list of errors related with missing alt text in makers are displayed

## Acceptance criteria
Like marker icons, marker shadows should have alternative text

**Comment by mourner** [comment]
I think marker shadow should have `alt=""` rather than copying the alt of the main marker, since it's a purely presentational element without semantical meaning, and should be treated as such by screen readers.

**Comment by topicus** [comment]
@mourner I've created a PR that does that https://github.com/Leaflet/Leaflet/pull/5259

**Comment by luiscarloscb** [comment]
I agree with @mourner, researching a bit about the best accessibility practices for alt tags in "decorative" images, I found many recommendations to leave the tag empty, if only to avoid clutter for assistive tools like screen readers.

For example, there's the [Web Accessibility Initiative recommendations for decorative images](https://www.w3.org/WAI/tutorials/images/decorative/), which provides some examples as well.

**Comment by topicus** [comment]
@mourner @luiscarloscb I've updated the PR to leave shadow's alternative text attribute empty.
